pyanaconda/ui/gui/spokes/blivet_gui.py

```diff
@@ -141,6 +139,7 @@ def refresh(self):
 
         self._back_already_clicked = False
 
+        config.default_fstype = self._storage.default_fstype
```

Comment:
@vojtechtrefny, can I move this line here?

Reply:
Yes, this should be save. blivet-gui uses this value in the add and edit dialogs so it's not picked up before running `refresh`.